changelog/20425.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+database/mongodbatlas: Add support for client certificate credential type
```

Comment:
There is a [special format](https://hashicorp.atlassian.net/wiki/spaces/VAULT/pages/1311244491/Changelog+Process#New-and-Major-Features) for features in the changelog. 

Suggest something like:
```suggestion
**MongoDB Atlas Database Secrets**: Adds support for client certificate credentials
```